docs/application/native/guides/app-management/common-appcontrol.md

```diff
@@ -2246,7 +2246,7 @@ From Tizen 5.0, the privacy setting guide popup application control is supported
 
 ### Shows privacy related common guide popup
 
-To show common privacy setting guide popup, use the `APP_CONTROL_OPERATION_PRIVACY_SETTING_GUIDE` operation. This operation helps you to guide users to change privacy privilege settings. After using this application control, check permissions to verify whether the user granted the requested permissions.
+To show common privacy setting guide popup, use the `APP_CONTROL_OPERATION_PRIVACY_SETTING_GUIDE` operation. This operation helps you to guide users to change privacy privilege settings. After using this application control, check permissions to verify whether the user granted the requested permissions. This app control's launch_mode is 'group'. When calling this app control, the caller should care about the app control's life cycle.
```

Comment:
For, "After using this application control, check permissions to verify whether the user granted the requested permissions."
Please change it to:
"After using this application control, check permissions to verify whether the user has granted the requested permissions."


This line: "This app control's launch_mode is 'group'. When calling this app control, the caller should care about the app control's life cycle."
can we rephrase it to:
The launch mode of this application control is *group*. The launch mode of this application control is *group*. While calling this control, you must be aware about the app control life cycle.

I have a question here:
About what the user must be aware?
What is the issue caused if the caller is not paying attention to the life cycle?

Reply:
Hello, Reni-90

I think we should keep `launch_mode` in this form as yl33 wrote.
Please refer [here](https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/guides/app-management/app-controls.md#application-group-management). In the article, it is also guiding [how to set launch mode in the manifest file](https://github.com/Samsung/tizen-docs/blob/master/docs/application/tizen-studio/native-tools/manifest-text-editor.md) like below.

![image](https://user-images.githubusercontent.com/9958374/42926210-aa463382-8b6b-11e8-8ba5-620864e06129.png)

Also, I hope [this guide](https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/tutorials/feature/best-practice-battery.md#app_battery_lifecycle) or [this one](https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/guides/app-management/ui-app/efl-ui-app.md#application-states-and-transitions) can help you to understand.

However, is it common to use "aware about" in the guide?
Also, the same sentence is duplicated in your review result.
"*The launch mode of this application control is group.* The launch mode of this application control is group. While calling this control, you must be aware about the app control life cycle."

Reply:
This app control is launched as group with caller app. It means that the app control will not be terminated until the explicit sign of termination is given. Explicit signs are clicking h/w back key, clicking buttons on the app control, terminated by caller, or so. 

For example, user can press home key while the app control is launched and the app is lowered. Of course the app control goes down with the app. In recent app menu, the caller app's topmost screen is the app control. And when the app is resumed, app might try to call app control again thinking the app control is terminated by the home key input because they didn't launch the app control as group mode explicitly. If that kind of call occurs again and again then the app control process can be multiply and never terminate until the caller die. SO, I mentioned it on application control description to notify app developers can manage it properly.

Reply:
Yes we should keep launch_mode in this form as yl33 wrote. I am sorry.
I was rephrasing it in a word file, so got it wrong.
I was trying to write :
The launch_mode of this application control is group. While calling this control, you must be aware about the app control life cycle."

Reply:
@Reni-90 
In fact, I'm familiar with "aware of", but not so with "aware about".
So "aware about" sounds tough to me, thus I can't sure with it..

Also I include the search result about the "aware about" and "aware of"
https://forum.wordreference.com/threads/aware-of-aware-about.911804/
https://www.thesaurus.com/browse/be%20aware%20of
https://www.italki.com/question/368407

Would you please confirm that it is really ok, to use "aware about", not "aware of"?

Reply:
@mijin85cho 
Thank you so much for pointing out. A new learning for me as well. 
I think 'aware of' is better than 'aware about'.
So, will rephrase to:
The launch_mode of this application control is group. While calling this control, you must be aware of the app control life cycle."

Reply:
Updated reviewed phrases.

Reply:
@yl33 
Thank you.